crates/ironclaw_gateway/static/styles/surfaces/chat.css

```
@@ -1573,6 +1686,676 @@
+  color: var(--success);
+  opacity: 0;
+  animation: flowChipIn 240ms var(--ease-out-expo) forwards;
+}
+
+.flow-cascade-via {
+  font-size: 10px;
+  font-weight: 500;
+  color: var(--text-dimmed);
+}
+
+@keyframes flowChipIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "flowChipIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)